staging/src/k8s.io/apiserver/pkg/util/flowcontrol/apf_controller.go

```diff
@@ -605,16 +604,15 @@ func (cfgCtlr *configController) digestConfigObjects(newPLs []*flowcontrol.Prior
 
 		// if we are going to issue an update, be sure we track every name we update so we know if we update it too often.
 		currResult.updatedItems.Insert(fsu.flowSchema.Name)
-		if klogV := klog.V(4); klogV.Enabled() {
-			klogV.Infof("%s writing Condition %s to FlowSchema %s, which had ResourceVersion=%s, because its previous value was %s, diff: %s",
-				cfgCtlr.name, fsu.condition, fsu.flowSchema.Name, fsu.flowSchema.ResourceVersion, fcfmt.Fmt(fsu.oldValue), cmp.Diff(fsu.oldValue, fsu.condition))
-		}
+		logger.V(4).Info("Writing Condition to FlowSchema", "controller", cfgCtlr.name,
+			"flowSchema", fsu.flowSchema.Name, "condition", fsu.condition, "oldValue", fsu.oldValue, "newValue", fsu.condition)
```

Comment:
(A) Showing the old value and the new value is more verbose than showing the old value and the diff.

(B) Why stop showing the ResourceVersion?

The design here was intended to help the reader identify and understand controller fights over these objects.

Reply:
Readded RV. Multiline log output is noise. We should avoid it. It dominates the log output of a startup today, and it repeatedly confuses ppl thinking this is an abnormal condition or reason for integration test failures.

Reply:
I see. In that case, I am OK with this change.

Reply:
Am curious where @s-urbaniak is getting to in his work to replace go-cmp use from the repo. Diffs have value, especially when the objects are bigger than just a condition like here. Wondering whether there is some sensible way to show a diff in a line. JSON Patch could. But that's also ugly as hell.

Reply:
(not a blocker on this PR)